ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/constants/CounterConstants.java

```diff
@@ -0,0 +1,6 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.constants;
+
+public class CounterConstants {
```

Comment:
Can you add this in the fragmentisers-common module? We try to keep the domain module as small as possible.